pkgs/by-name/cd/cdecl/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bison,
+  flex,
+  readline,
+  ncurses,
+  gcc
+}:
+
+stdenv.mkDerivation {
+  pname = "cdecl";
+  version = "2.5-unstable-2024-05-07";
+
+  src = fetchFromGitHub {
+    owner = "ridiculousfish";
+    repo = "cdecl-blocks";
+    rev = "1e6e1596771183d9bb90bcf152d6bc2055219a7e";
+    hash = "sha256-5XuiYkFe+QvVBRIXRieKoE0zbISMvU1iLgEfkw6GnlE=";
+  };
+
+  buildInputs = [
+    gcc
+    bison
+    flex
+    readline
+    ncurses
+  ];
```

I guess it will be more like this:

```suggestion
  strictDeps = true;

  nativeBuildInputs = [
    bison
    flex
  ];

  buildInputs = [
    readline
    ncurses
  ];
```

Also pass `"CC=${stdenv.cc.targetPrefix}cc"` to `makeFlags`  if necessary.

(I can't test, though, I'm on a train from Paris to Cannes with terrible connectivity)

---

seems to work without `"CC=${stdenv.cc.targetPrefix}cc"`
What does `strictDeps = true;` do?

---

*Hello follow french nixer :o*

---

It broke for darwin, trying `"CC=${stdenv.cc.targetPrefix}cc"`